src/js/services/bitpayService.js

```diff
@@ -0,0 +1,193 @@
+'use strict';
+
+angular.module('copayApp.services').factory('bitpayService', function($log, $http, platformInfo, appIdentityService, bitauthService, storageService, gettextCatalog, popupService) {
+  var root = {};
+
+  var NETWORK = 'livenet';
+  var BITPAY_API_URL = NETWORK == 'livenet' ? 'https://bitpay.com' : 'https://test.bitpay.com';
+
+  root.getEnvironment = function() {
```

Comment:
To match the rest of Copay's code, this would probably be either: getNetwork
or getEnvironment and return `{ network: "livenet" }`.